src/lint/rules/tests/spacing/mod.rs

```diff
@@ -32,11 +31,11 @@ bank pcie_config {register command {field mem {
 fn style_check_sp_braces() {
     let mut cfg = LintCfg::default();
     let mut rules = instantiate_rules(&cfg);
-    assert_snippet(SP_BRACES, 8, &rules);
+    assert_snippet(SP_BRACES, 9, &rules);
     // Test rule disable
     cfg.sp_brace = None;
     rules = instantiate_rules(&cfg);
-    assert_snippet(SP_BRACES, 0, &rules);
+    assert_snippet(SP_BRACES, 1, &rules);
```

Comment:
This is testing the disable of the rule. Expected value should be 0 and no errors should be reported. If an error is reported, it is caused by another rule that should not be tested in this snippet.